feature-student/wakeup-song/src/main/java/com/b1nd/dodam/wakeupsong/navigation/OutingNavigation.kt

```diff
@@ -0,0 +1,22 @@
+package com.b1nd.dodam.wakeupsong.navigation
+
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavOptions
+import androidx.navigation.compose.composable
+import com.b1nd.dodam.wakeupsong.WakeupSongScreen
+
+const val WAKEUP_SONG_ROUTE = "wakeup-song"
```

Comment:
Change the name of the root from `wakeup-song` to `wakeup_song`. That's not following convention.